prime_factor/prime_factor.py

```diff
@@ -0,0 +1,39 @@
+import math
+
+
+def prime_factor(num):
+    """
+
+    Prime factor of a number
+
+
+    :param start : checking number if it is a prime factor of number entered
+    :param counter : checking the number of factors of start
+    :param checker : checks if start is a prime number or not
+    :return : no returns
+    """
+    print("Factors are:")
```

Comment:
No need to `print` here (in this function). 
Return a list of numbers and make the `main` function print.